src/utils.js

```diff
@@ -126,7 +126,7 @@ export function findValue (obj, valuePath, startPath = '') {
   const parentLevels = valueLevels.filter((element) => element === '.')
   const valueKey = valueLevels.pop()
   const absValuePath = _.without(depths.slice(0, depths.length - parentLevels.length - 1), '', '.').join('.')
-  const absValueKey = [absValuePath, valueKey].join('.')
+  const absValueKey = [absValuePath, valueKey].join('.').replace(/^\./, '')
```

Comment:
Newer version of lodash was choking here because the value end up being something like `.foo.bar[1].baz`. Simply removing the period at the beginning when it is present.